### ISSUE TITLE
Tab order for owner dashboard

### ISSUE THREAD

**Issue by saracarl** [benwbrum/fromthepage#367]
With the additon of user data to the owner dashboard, we should think about tab order and which tab should be "on" when you go to the owner dashboard.
the statistics page does say how much has been done in the last 7 days, but you can't get more specific

may depend on state of your works and collections.
